doc/create_ramp_event.rst

```diff
@@ -53,7 +53,27 @@ This config file should look like::
       more precisely the :ref:`conda workers <conda_env_worker>` to have more
       information.
 
-Finally, you can easily deploy the event (adding both problem and event to the
+Before you continue make sure that::
+    1.  you have the private and the public data in the following directory
+        structure::
```

Comment:
I think you need an empty line after the `::`

Also I think it would be better to have the bullet points outside of the 'code' block